spring-boot-autoconfigure/src/test/java/org/axonframework/springboot/JdbcAutoConfigurationTest.java

```diff
@@ -56,39 +53,38 @@
  *
  * @author Milan Savic
  */
-@ExtendWith(SpringExtension.class)
-@EnableMBeanExport(registration = RegistrationPolicy.IGNORE_EXISTING)
-@ContextConfiguration(classes = JdbcAutoConfigurationTest.Context.class)
-@EnableAutoConfiguration(exclude = {
-        JpaRepositoriesAutoConfiguration.class,
-        HibernateJpaAutoConfiguration.class,
-        AxonServerAutoConfiguration.class
-})
 public class JdbcAutoConfigurationTest {
 
-    @Autowired
-    private ApplicationContext applicationContext;
-
     @Test
-    void testContextInitialization() {
-        assertNotNull(applicationContext);
-
-        assertTrue(applicationContext.getBean(EventStorageEngine.class) instanceof JdbcEventStorageEngine);
-        assertTrue(applicationContext.getBean(PersistenceExceptionResolver.class) instanceof JdbcSQLErrorCodesResolver);
-        assertTrue(
-                applicationContext.getBean(ConnectionProvider.class) instanceof UnitOfWorkAwareConnectionProviderWrapper
-        );
-        assertTrue(applicationContext.getBean(TokenStore.class) instanceof JdbcTokenStore);
-        assertTrue(applicationContext.getBean(SagaStore.class) instanceof JdbcSagaStore);
+    void testAllJdbcComponentsAutoConfigured() {
```

Comment:
Why have the `JdbcSQLErrorCodesResolver` and `UnitOfWorkAwareConnectionProviderWrapper` assertions been removed from the test?

Reply:
But, they're right there ;-)

Reply:
Sure they are :p